packages/make/4.2.1/version.desc

```diff
@@ -1,2 +0,0 @@
-obsolete='yes'
-archive_formats='.tar.bz2 .tar.gz'
```

Comment:
I have run into issues on some open source project still not 100% compatible with make >= 4.3, perhaps we want to hold on for one more release?

Reply:
Sure. I'll drop this from the list.